mshadow/cuda/tensor_gpu-inl.cuh

```diff
@@ -6,6 +6,11 @@
  */
 #ifndef MSHADOW_CUDA_TENSOR_GPU_INL_CUH_
 #define MSHADOW_CUDA_TENSOR_GPU_INL_CUH_
+#include <thrust/device_ptr.h>
```

Comment:
Yes, thrust is included in cuda toolkit. Also, since it's a template library, there is no need to install. http://docs.nvidia.com/cuda/thrust/#axzz4EPrET26A . (I find that thrust has been added to CUDA toolkit long ago, maybe since CUDA 5 -- http://developer.download.nvidia.com/compute/cuda/5_0/rel/docs/CUDA_Toolkit_Release_Notes_And_Errata.txt).